pages/vi/profiles/Emiton.md

```diff
@@ -0,0 +1,42 @@
+# Profile: Emiton Alves  
+* Time Zone: Eastern 
+* OS: Windows 10 / Ubuntu 16.04
+* Location: Providence, RI
+
+![Providence, RI](https://c.o0bg.com/rf/image_960w/Boston/2011-2020/2016/05/17/BostonGlobe.com/Travel/Images/ProvidenceA2.jpg)
+> My hometown
+
+## About Me:
+I am a software engineering student with a background in Mechanical Engineerinng. I was introduced to programming my senior year and became interested in learning on my own. After taking a few courses and teaching myself during my free time, I was able to enroll in a Master's program for CS. In my freetime I am currently working with web technologies like MEAN stack and also building web applicatons using C#.
+
+Add me on **LinkedIn** :arrow_right: [LinkedIn](https://www.linkedin.com/in/emiton-alves-4b418672/)
```

Comment:
Emojis don't work with rawgit; you might want to replace it with something else. Everything else looks great! Nice use of markdown features :)